src/main/scala/com/typesafe/sbt/packager/linux/Keys.scala

```diff
@@ -13,6 +13,7 @@ trait LinuxKeys {
   val daemonGroup = SettingKey[String]("daemon-group", "Group to start application daemon")
   val daemonGroupGid = SettingKey[Option[String]]("daemon-group-gid", "GID of daemonGroup")
   val daemonShell = SettingKey[String]("daemon-shell", "Shell provided for the daemon user")
+  val daemonLogFile = SettingKey[String]("daemon-log-file", "Name of the log file generated by the application daemon")
```

Comment:
for what reason? Not sure if other packages/system start daemons may use this as well. debian systemv for example.